README.md

```diff
@@ -25,21 +27,24 @@ cmake --build build
 ctest --test-dir build
 ```
 
-### `OPENASSETIOTEST_ENABLE_SUBPROJECT=OFF` (default)
+### ENABLE_SUBPROJECT=OFF (default)
```

Comment:
[question] Is this meant to be suggesting there is a var called `ENABLE_SUBPROJECT`? If its meant to be a shorthand for either of the individual dependency vars then it's perhaps a little misleading using that exact presentation. It looks the same as a real var, so made me go look to see if we had introduced an unprefixed var etc...  Perhaps a normal style heading about subproject use vs var + default.